src/app/components/dashboard-apps-dock/dashboard-apps-dock.less

```diff
@@ -0,0 +1,304 @@
+@mno-dock-add-app-border-color: white;
+@mno-dock-add-app-text-color: white;
+@mno-dock-expand-arrow-color: white;
+@mno-dock-expand-arrow-color-hover: darken(@mno-dock-expand-arrow-color, 5%);
+@mno-dock-expand-left-border-color: white;
+@mno-dock-bg-color: @decorator-inverse-color;
+@mno-dock-app-title-color: white;
+
+@mno-dock-sub-menu-bg-color: @darkblue;
+@mno-dock-sub-menu-text-color: white;
+@mno-dock-sub-menu-icon-color: @green;
+
+@mno-dock-app-logo-square-width: 70px;
+@mno-dock-app-logo-minimized-square-width: 50px;
```

Comment:
Move those variables in variables.less. It will then be easy for the host project to know what variables are available.